dmriprep/data/tests/dwi.tsv

```diff
@@ -0,0 +1,106 @@
+# R	A	S	B
```

Comment:
A rasb file. You can assume this file is kosher because the bids-validator will make all the hard work.